### ISSUE TITLE
setup.py formatting issues

### ISSUE THREAD

**Issue by ludaavics** [dephell/dephell#443]
Dephell for setup.py generation and code formatters (e.g. [black](https://github.com/psf/black)) can't be used concurrently as [pre-commit](https://pre-commit.com/) hooks.

```toml
# pyproject.toml
# .. poetry config ..
[tool.dephell.main]
from = {format = "poetry", path = "pyproject.toml"}
to = {format = "setuppy", path = "setup.py"}
```

```yaml
# .pre-commit-config.yaml
repos:
- repo: https://github.com/psf/black
  rev: 19.10b0
  hooks:
  - id: black
    name: Blacken Python files
    args:
    - --check
- repo: https://github.com/mverteuil/precommit-dephell
  rev: master
  hooks:
    - id: dephell
```
If i try to commit after a `dephell deps convert`, the code formatter check fails.
If I try to format the generated `setup.py`, the dephell pre-commit hook fails.

As a work around, I have excluded `setup.py` from the formatter's pre-commit check:

```yaml
- repo: https://github.com/psf/black
  rev: 19.10b0
  hooks:
  - id: black
    name: Blacken Python files
    args:
    - --check
    exclude: setup.py
```

But it would be nice to have it both ways! Perhaps leave setup.py intact if the AST is unchanged?

Thanks!
# Versions

```json
{
  "cache": "0b",
  "path": "/home/ltiako/.local/share/dephell/venvs/dephell/lib/python3.8/site-packages/dephell",
  "python": "/home/ltiako/.local/share/dephell/venvs/dephell/bin/python3",
  "version": "0.8.3",
  "versions": {
    "dephell-archive": "0.1.6",
    "dephell-argparse": "0.1.2",
    "dephell-changelogs": "0.0.1",
    "dephell-discover": "0.2.10",
    "dephell-licenses": "0.1.6",
    "dephell-links": "0.1.5",
    "dephell-markers": "1.0.3",
    "dephell-pythons": "0.1.14",
    "dephell-setuptools": "0.2.3",
    "dephell-shells": "0.1.4",
    "dephell-specifier": "0.2.2",
    "dephell-venvs": "0.1.17",
    "dephell-versioning": "0.1.1"
  }
}
```

**Comment by orsinium** [comment]
Thank you for the feedback.

DepHell supports yapf and autopep8:
https://github.com/dephell/dephell/blob/master/dephell/converters/setuppy.py#L23-L33

If you need to use another autofomatter, possible solutions I see:
1. Write your own pre-commit hook
1. Open an issue in `mverteuil/precommit-dephell` to extend the existing hook or add another one next to it.
1. Google how to chain hooks in pre-commit
1. Don't run them in parallel.
1. Exclude the generated files. There is not much point in checking them anyway.

Anyway, I don't think it's a good idea to add support for every existing formatter into dephell itself. Everyone has a different tooling and style guides, we can't handle them all.
